flask/src/main.py

```diff
@@ -96,6 +99,20 @@ def __init__(self, import_name, *args, **kwargs):
 app = MyFlask(__name__)
 CORS(app)
 
+cache_config = {
+    "DEBUG": True,          # some Flask specific configs
+    "CACHE_TYPE": "RedisCache",  # Flask-Caching related configs
+    "CACHE_DEFAULT_TIMEOUT": 300
+}
+
+redis_host = os.environ.get("REDISHOST", "localhost")
```

Comment:
just curious, why can't it make a direct connection?

Reply:
Ah got it. Then when running in Google App Engine you shouldn't need it right? Assuming everything is configured to be within same VPC
Not sure if it's similar but fwiw GAE does some trickery under the hood to connect to Postgres (for most environments/languages it's done automatically under the hood but on some needs manual config): https://cloud.google.com/sql/docs/postgres/connect-auth-proxy

Reply:
> An improvement later could be to migrate it to Google Compute Engine VM, then it will able to connect directly: https://cloud.google.com/memorystore/docs/redis/connect-redis-instance-gce

nah, let's not do that. GAE is easier to manage and deploy to